frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -64,12 +64,33 @@ export const Body = styled.main`
   position: relative;
 `;
 
-export const Footer = styled.footer`
+export const ActionButtonsContainer = styled.div`
+  color: ${color("text-medium")};
+  margin-left: auto;
+`;
+
+const footerVariantStyles = {
+  default: css`
+    border-top: 1px solid ${color("border")};
+  `,
+  big: css`
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 2rem;
+
+    ${ActionButtonsContainer} {
+      display: none;
+    }
+  `,
+};
+
+export const Footer = styled.footer<{ variant: "default" | "big" }>`
```

Comment:
Saw `Variant` is being exported in `frontend/src/metabase/public/components/EmbedFrame/LogoBadge.styled.tsx`. Should we reuse that?